resources/assets/icons/brands/outline/twitter.svg

```diff
@@ -1 +1 @@
-<svg id="Regular" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M23.18 6.71l-2.1-.9 1.1-2.4-2.56.85a4.47 4.47 0 00-3-1.2 4.51 4.51 0 00-4.5 4.5v1c-3.54.73-6.63-1.2-9.5-4.5q-.75 4 1.5 6l-3.29-.5a4.55 4.55 0 004.25 4l-2.75 1c1 2 2.82 2.31 5.25 2.5a11.52 11.52 0 01-6.75 2c12.76 5.67 20.25-2.66 20.25-10v-.83z" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="16.846" height="22.183"><g id="Twitter" transform="translate(-418.733 -298.754)"><g id="Group_175" data-name="Group 175" transform="translate(419.733 299.754)"><g id="Regular_175" data-name="Regular 175"><path id="Shape_512" data-name="Shape 512" d="M0 2.691a2.691 2.691 0 015.382 0v1.794h6.773a2.691 2.691 0 110 5.382H5.382v2.243a2.691 2.691 0 002.691 2.691h4.082a2.691 2.691 0 010 5.382H7.176A7.177 7.177 0 010 13.007z" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/></g></g></g></svg>
```

Comment:
this is used on ark.io and others:
<img width="447" alt="image" src="https://user-images.githubusercontent.com/35610748/105711046-45cf4f00-5f18-11eb-893e-ec0d8b21c8cc.png">

you sure we want to have this updated to the `t` character? 🤔

Reply:
I shall remove zis